src/main/java/schulmanager/components/ChatOverviewBox.java

```diff
@@ -71,15 +80,28 @@ public ChatOverviewBox(JsonObject subscription) {
 		senderSpan.getStyle().set("text-overflow", "ellipsis").set("white-space", "nowrap").set("overflow-x", "hidden").set("width", "100%").set("color", "var(--lumo-contrast-80pct)").set("font-size", "14px");
 		recipientSpan.getStyle().set("text-overflow", "ellipsis").set("white-space", "nowrap").set("overflow-x", "hidden").set("width", "100%").set("color", "var(--lumo-contrast-80pct)").set("font-size", "14px");
 		textPart.add(subjectSpan, senderSpan, recipientSpan);
-		
+
+		// VerticalLayout to the right that contains the context menu button and the timestamp of last message
 		VerticalLayout optionPart = new VerticalLayout();
 		optionPart.setPadding(false);
 		optionPart.setSpacing(false);
-		Span lastMessageSpan = new Span(beautifyLocalDateTime(lastMessageTimestamp));
+
+		// the date at the top right
+		Span lastMessageSpan = new Span(beautifyLocalDateTime(lastMessageTimestamp) + "  ");
```

Comment:
Um... Correct me, if I'm wrong, but I always thought, HTML removes trailing spaces.

Reply:
looks like it doesnt
![2024-06-05_18-42](https://github.com/IllusioQuest/teacher-quotation-manager/assets/95178340/a9fba64a-de29-485e-9c90-0883bfa4e57b)

Reply:
Sorry to disappoint you, but I just tested again and trailing whitespaces don't affect anything. The new position is just because you added a margin on the right...